R/explanation.R

```diff
@@ -603,7 +603,7 @@ create_S_batch <- function(explainer, n_batches, index_features = NULL) {
 
   if (!is.null(index_features)) {
     # Rescale the number of batches to the percentage of observations used
-    n_batches <- floor(length(index_features) / nrow(explainer$S) * n_batches)
+    n_batches <- max(1,floor(length(index_features) / nrow(explainer$S) * n_batches))
```

Comment:
@JensWahl Please ensure this fix does not break anything else. This caused a problem as it returned n_batches = 0, not 1 when used with a combination approach.

Reply:
Thanks, I actually fixed this issue on a local test branch, but didn't push ut.